### DISCUSSION TITLE
OSX Buil failure with error:"Undefined symbol: matplot::gca()"

### DISCUSSION THREAD

**Discussion by re-tart** [alandefreitas/matplotplusplus#106] [Q&A]
I just installed matplot++ (on my osx10.15.4) and tried a simple `plot(x,y)` and ended up with build failure with error 

```
"Undefined symbol: matplot::gca()
```

,then i tried using `gca()` directly and ended up with exactly the same error.

I have included my `<matplot/matplot.h>`  header file as instructed , now i have no clue what could be the cause of this error. 

Any help would be appreciated

**Comment by alandefreitas** [comment]
You need to link the library.

**Reply by re-tart** [reply]
shoot.thanks.

**Reply by re-tart** [reply]
ehh.and where can i get the name of the library? or can you just tell me? as you can tell, i am a newbie in coding hhhhh

**Reply by re-tart** [reply]
so now i found and linked libmatplot.dylib,but now i got a runtime error:sh: gnuplot: command not found. how can i fix this problem?

**Reply by re-tart** [reply]
problem solved!!thanks anyway
